Auth/AuthViewController.swift

```diff
@@ -1,34 +1,35 @@
 import UIKit
-
-protocol AuthViewControllerDelegate: AnyObject {
-    func authViewController(_ vc: AuthViewController, didAuthenticateWithCode code: String)
+protocol AuthViewDelegate: AnyObject {
+    func authViewcontroller (_ vc: AuthViewController, didAuthenticateWithCode code: String)
```

Comment:
```suggestion
    func authViewController(_ vc: AuthViewController, didAuthenticateWithCode code: String)
```

Давай договоримся: 
1) все слова писать в camelCase(вердблюжий стиль), здесь у Controller надо было с большой буквы. функции, переменные, свойства и константы - начинаются с маленькой буквы, затем все слова с большо
2) перед круглой скобкой пробел не ставится